web/beacon-app/cypress/e2e/OnboardingInvitationUser.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Invited User Onboarding
+
+I want to complete onboarding after receiving an invitation to join an organization.
+
+Scenario: Onboarding for a new invited user
+
+Given I'm on the login page
+When I log into Beacon
+Then I should be directed to the onboarding form
+And I should see the onboarding sidebar
+And I should see the name of the team I have been invited to join
+And I should see my email address
+And I should see the option to log out
+And I should see step 3 of the onboarding form
+When I click the Back button on the third step of the onboarding form
+Then I should be directed to the second step of the onboarding form
+And I should not be able to edit the workspace URL
+# When I click the Back button on the second step of the onboarding form
```

Comment:
This is testing the happy path for now. I'll create a follow on story to add the commented out steps back to the test.